src/main/steps/urls.ts

```diff
@@ -155,6 +155,9 @@ export const DOWNLOAD_DOCUMENT: PageLink = `${DOCUMENTS}/download/:documentId/:d
 export const DOWNLOAD_DOCUMENT_BY_TYPE: PageLink = `${DOCUMENTS}/download/type/:documentType/:forceDownload?`;
 
 export const RESPONDENT_TO_APPLICATION_SUMMARY: PageLink = '/tasklistresponse/summary';
+export const RESPONDENT_TO_APPLICATION_SUMMARY_REDIRECT: PageLink = '/tasklistresponse/summary/equality';
```

Comment:
is this required?

Reply:
Added previously for multiple routes, Removed now